### PR TITLE
[docs] add note about mtls certgen cronjob

### PR DESCRIPTION
**Pull request by jenshu**
add note about mtls certgen cronjob

### CODE REVIEW

**Review by sam-heilbron — COMMENTED** [review]
LGTM! I'll leave open if the docs team has any suggestions.

File: docs/content/guides/security/tls/mtls/_index.md [hunk 1]
```diff
@@ -244,6 +244,19 @@ The configuration for the extauth envoy sidecar can be found in the rate-limit-s
 Cert rotation can be done by updating the gloo-mtls-certs secret. The SDS sidecar will
 automatically pick up the change.
 
+Automatic cert rotation can be enabled via a CronJob by setting the Helm value `gateway.certGenJob.cron.enabled=true` at install time:
```
Comment (Rachael-Graham) [inline docs/content/guides/security/tls/mtls/_index.md:247]:
```suggestion
If you want to automatically rotate certs based on a schedule, you can enable a cert rotation CronJob. In your Helm values file for Gloo Edge installation, set `gateway.certGenJob.cron.enabled` to `true`, and specify a rotation schedule:
```

**Review by Rachael-Graham — COMMENTED** [review]
docs

**Review by Rachael-Graham — APPROVED** [review]

**Pull request merged**